doc/en/user/source/data/raster/imagemosaic/tutorial.rst

```diff
@@ -318,13 +318,13 @@ Multi-resolution imagery with reprojection
 
 As a general rule, we want to have the highest resolution granules shown "on top", with the lower-resolution granules filling in the gaps as necessary.
 
-In this example, we will serve up overlapping granules that have varying resolutions. In addition, we will mix resolutions, such that the higher resolution granule is reprojected to match the resolution of the 
+In this example, we will serve up overlapping granules that have varying resolutions. In addition, we will mix resolutions, such that the higher resolution granule is reprojected to match the resolution of the lower resolution granules.
```

Comment:
This sentence was previously incomplete, so I have guessed here about the correct end. I hope I got it right.